deployments/properties/aks.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Deploy to a AKS Cluster",
+    "description": "Deploy an application to a Azure Kubernetes Service Cluster using Azure Credentials",
+    "creator": "Microsoft Azure",
+    "iconName": "azure",
+    "categories": ["Deployment", "Kompose", "Helm", "Kustomize", "Kubernetes, "Dockerfile"]
```

Comment:
@gambtho 
1.  should we add Manifest to this list here ? 
2. The properties - Kompose, Helm, Kustomize, Kubernetes - are not being detected today. Can you update the detection logic to add those. And remove the properties from here which we cant detect ?  
3. the other template  - https://github.com/actions/starter-workflows/pull/1206/files  will always conflict with this one. I recommend to use only one of the templates. My understanding is that the two templates are differing wrt to authentication, if supporting both auth is really important, you might want to evaluate having one as default and other as comment. 
cc - @bishal-pdMSFT / @anuragc617

Reply:
- i added a PR to detect 3 of the 4...haven't seen any response to them
- happy to close the other template
- no idea how you'd want to detect manifest -- feel free to add detection logic, and i'll remove the other

- is it possible to get a consolidated PR review, rather than one-off comments every couple days?

Reply:
links to the detection PRs - https://github.com/actions/starter-workflows/pull/1207#discussion_r745551029
(in the comments above, from last week)

Reply:
@pulkitaggarwl what's the best way to have one as a comment?   have it hosted in a separate repo, and link to it?

Reply:
I would suggest adding some comments and routing them to documentation pages with details around second one

Reply:
sorry -- i'm not following.   Adding comments to the properties file, and routing them to where?

Reply:
@gambtho, sure I can elaborate. 
My suggestion was to add some comments in the 1st YML file highlighting that there is an alternate  CI / CD to AKS can be used.  An azure docs page with the alternate version and details of the YML components of the alternate version can be used.

Reply:
ahah, makes sense to me.  i'll close the other one for now, and will submit a new PR once we have a docs home for it